tests/test_hooks.py

```diff
@@ -181,21 +182,39 @@ def test_run_failing_hook(self):
             assert 'Hook script failed' in str(excinfo.value)
 
 
-@pytest.fixture
-def hook_backup_files():
-    """Return basenames of all files in the hooks dir."""
-    return sorted([
-        os.path.basename(f) for f in
-        os.listdir('tests/hooks-backup-files/hooks')
-    ])
+@pytest.yield_fixture
+def hook_backup_dir(tmpdir):
+    """Yield a directory that contains hook backup files."""
 
+    hooks_dir = tmpdir.mkdir('hook-files')
+
+    pre_hook_content = textwrap.dedent(
+        u"""
+        #!/usr/bin/env python
+        # -*- coding: utf-8 -*-
+        print('pre_gen_project.py~')
+        """
+    )
+    pre_gen_hook_file = hooks_dir / 'pre_gen_project.py~'
```

Comment:
Good catch! 

In the long run we probably want to change the code to accept a path rather than using the current working directory + ``hooks``.